src/lib/components/TableToolBarModal/TableToolBarModal.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button } from '../commons/Button';
+import { StyledModalTitle } from '../commons/Modal/Modal.styled';
+import { StyledTableToolbarModal } from './TableToolBarModal.styled';
+import { ITableToolBarModalProps } from './TableToolBarModal.types';
+import { useModalContext } from '@/lib/context/ModalProvider/ModalProvider';
+import { DISPLAY_TEXTS, EButtonTexts } from '@/lib/consts/displayTexts';
+export const TableToolBarModal = ({
+  texts,
+  action,
+}: ITableToolBarModalProps) => {
+  const buttonsText = DISPLAY_TEXTS.he.buttons;
+  const { display, button, title, type } = texts;
+  const modalTitle = display[button](display[type]);
+  const { closeModal } = useModalContext();
+  return (
+    <StyledTableToolbarModal>
+      <StyledModalTitle>{modalTitle}</StyledModalTitle>
+      <p className='modalText'>{display[`${button}Ensure`](title)}</p>
```

Comment:
what is this display ? if it's a function then make the name more meaningful , e.g. getDisplayText

Reply:
modalText might be named StyledBody

Reply:
done!